business_requirement_deliverable/report/br_deliverable_sale_report_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_deliverable_sale_graph" model="ir.ui.view">
+             <field name="name">Deliverable Sale Report</field>
+             <field name="model">business.requirement.deliverable.sale.report</field>
+             <field name="arch" type="xml">
+                 <graph string="Sales Analysis" type="pivot" stacked="True">
+                     <field name="name" type="row"/>
+                     <field name="br_count" type="measure"/>
+                     <field name="dlv_count" type="measure"/>
+                     <field name="res_count" type="measure"/>
+                     <field name="res_qty" type="measure"/>
+                     <field name="dlv_qty" type="measure"/>
+                     <field name="sale_price" type="measure"/>
+                     <field name="total_revenue" type="measure"/>
+                 </graph>
+             </field>
+        </record>
+
+        <record id="action_deliverable_sale_report" model="ir.actions.act_window">
+            <field name="name">Deliverable Sale Analysis</field>
+            <field name="res_model">business.requirement.deliverable.sale.report</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">graph</field>
+            <field name="view_id" ref="view_deliverable_sale_graph"/>
+        </record>
+
+        <menuitem action="action_deliverable_sale_report" id="menu_action_deliverable_sale_report"
```

Comment:
@darshan-serpent Please help reviewers. XML parameters should be place on one column, like this:

action="action_deliverable_sale_report" 
id="menu_action_deliverable_sale_report" 
parent="base.menu_project_report"
groups="business_requirement_deliverable.group_business_requirement_estimation"